adafruit_ads1x15/ads1x15.py

```diff
@@ -131,6 +165,61 @@ def gains(self) -> List[float]:
         g.sort()
         return g
 
+    @property
+    def comparator_queue_length(self) -> int:
+        """The ADC comparator queue length."""
+        return self._comparator_queue_length
+
+    @comparator_queue_length.setter
+    def comparator_queue_length(self, comparator_queue_length: int) -> None:
+        possible_comp_queue_lengths = self.comparator_queue_lengths
+        if comparator_queue_length not in possible_comp_queue_lengths:
+            raise ValueError(
+                "Comparator Queue must be one of: {}".format(
+                    possible_comp_queue_lengths
+                )
+            )
+        self._comparator_queue_length = comparator_queue_length
+
+    @property
+    def comparator_queue_lengths(self) -> List[int]:
+        """Possible comparator queue length settings."""
+        g = list(_ADS1X15_CONFIG_COMP_QUEUE.keys())
+        g.sort()
+        return g
+
+    @property
+    def comparator_low_threshold(self) -> int:
+        """The ADC Comparator Lower Limit Threshold."""
+        return self._comparator_low_threshold
+
+    @property
+    def comparator_high_threshold(self) -> int:
+        """The ADC Comparator Higher Limit Threshold."""
+        return self._comparator_high_threshold
+
+    @comparator_low_threshold.setter
+    def comparator_low_threshold(self, value: int) -> None:
+        """Set comparator low threshold value for ADS1015 ADC
+
+        :param int value: 16-bit signed integer to write to register
+        """
+        if value < 0 or value > 65535:
```

Comment:
Isn't this the right range to check against since it is signed?
```suggestion
        if value < -32768 or value > 32767:
```

Reply:
You are technically correct, but python will look at 0xFFFF as 65535 and say that it is out of range, instead of looking at it as -1 and say that it is in range. I feel like what I did was a workaround to make things work.

Reply:
I just discovered "a_bytes_big = an_int.to_bytes(2, 'big', signed = True)" that should convert between signed integers and bytes easier. I'll try to work that in.